src/components/molecules/ChatsList/ChatsList.tsx

```diff
@@ -170,15 +171,32 @@ const ChatsList: React.FunctionComponent = () => {
                     setSelectedUserProfile={noopHandler}
                   />
                   <div className="private-message-content">
-                    <div className="private-message-author">{profileName}</div>
-                    <div className="private-message-last">
+                    <div
+                      className={`private-message-author ${
+                        isUnreadMessage && "unread"
+                      }`}
+                    >
+                      {profileName}
+                    </div>
+                    <div
+                      className={`private-message-last ${
+                        isUnreadMessage && "unread"
+                      }`}
+                    >
                       {lastMessageExchanged.text}
                     </div>
                   </div>
-                  {lastMessageExchanged.from !== user?.uid &&
-                    !lastMessageExchanged.isRead && (
-                      <div className="not-read-indicator" />
-                    )}
+                  {lastMessageExchanged.from !== user?.uid && (
+                    <div
+                      className={`private-message-time ${
+                        isUnreadMessage && "unread"
+                      }`}
+                    >
+                      {formatDistanceToNow(
+                        new Date(lastMessageExchanged.ts_utc.toDate())
```

Comment:
if you're calling `.toDate()`, do you need to wrap this in `new Date()`?

- https://date-fns.org/v2.2.1/docs/formatDistanceToNow#arguments

It seems it can take a `Number` as it's parameter, so can we just pass `.ts_utc` directly?

Reply:
.ts_utc  is Timestamp that's why i'm callign .toDate() but I'll se if it works without new Date